flutter/news/lib/features/frontpage/domain/usecases/get_everything_about.dart

```diff
@@ -1,18 +1,14 @@
-import 'package:dartz/dartz.dart';
-import 'package:news/core/error/failures.dart';
-import 'package:news/core/usecases/usecase.dart';
+import 'package:news/core/result.dart';
 import 'package:news/features/frontpage/domain/entities/article.dart';
-import 'package:news/features/frontpage/domain/usecases/get_top_headlines.dart';
 
 import '../../data/repositories/articles_repository.dart';
 
-class GetEverythingAbout implements UseCase<List<Article>, Params> {
```

Comment:
I don't see the value (but I understand its a common practice) of having a generic use case.

I am happy to bring it back if someone can explain the value of having that thing to me. But please be more concrete than just `re-usability`